aligner/u_of_michigan_aligner/u_of_michigan_aligner.wdl

```diff
@@ -30,6 +30,15 @@ workflow TopMedAligner {
   File dbSNP_vcf
   File dbSNP_vcf_index
 
+  Int? PreAlign_CPUs
+  Int PreAlign_CPUs_default = select_first([PreAlign_CPUs, 2])
+
+  Int? Align_CPUs
+  Int Align_CPUs_default = select_first([Align_CPUs, 32])
```

Comment:
If `PreAlign_CPUs` isn't set, it will default to 32? But what happens if the machine has less than 32 cores?

Reply:
Yes if PreAlign_CPUs isn't set it will default to 32. Right now FireCloud uses Google Cloud compute which has machines with 32 CPUs. see here https://cloud.google.com/compute/pricing. I have a question in to FireCloud about what happens if no machine is available...

Reply:
Google will wait for a machine for 32 cpu's to become available, so the job shows up as "Running" in the UI, and behind the scenes its "Waiting for Quota"

Reply:
Sounds good. I suppose it's always better to wait rather than go down in the number of cores should a machine with less cores be available?

Go ahead and squash/merge.